pandas/_libs/tslibs/offsets.pyx

```diff
@@ -4762,6 +4763,13 @@ cpdef to_offset(freq, bint is_period=False):
                         f"instead of \'{name}\'"
                     )
                 elif is_period is True and name in c_OFFSET_DEPR_FREQSTR:
+                    if name.startswith("A"):
+                        warnings.warn(
+                            f"\'{name}\' will be deprecated, please use "
```

Comment:
can we change "will be deprecated" to "is deprecated and will be removed in a future version"?

Reply:
yes, I changed the warning message and updated my PR.